### PR TITLE
Update screenshots in user guide with latest UI

### PR DESCRIPTION
**Pull request by rukekre**
## What was the problem/requirement? (What/Why)
The user doc contained certain UI images that were older and didn't include newer functionality.

## What was the solution? (How)
Relevant screenshots have been updated in the user guide to match the latest UI. Changes have been made to the text to include any new information.

## What artifacts are related to this change?
Issues: (internal) P54511997

## What is the impact of this change?
Any users referring to the user guide will be provided with the latest information.

## Are you adding any new dependencies to the system?
No.

## How were these changes tested?
Locally, using the standard procedure to view changes to the user doc.

----
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by haozhao-aws — COMMENTED** [review]
I am done with this round review. Only have one question about the order of the static obstacles.

File: docs/user-guide/mkdocs.yml [hunk 1]
```diff
@@ -6,10 +6,10 @@ nav:
         - Setup: setup.md
     - Static Obstacles:
         - Overview: spawner-parent.md
-        - Spawn on Surface: spawn-on-surface.md
         - Spawn in Volume: spawn-in-volume.md
-        - Spawn with Houdini Digital Assets: spawn-with-houdini.md
         - Spawn on Path: spawn-on-path.md
+        - Spawn on Surface: spawn-on-surface.md
```
[the remark this answers is not in the document]
Comment (rukekre) [reply docs/user-guide/mkdocs.yml:11]: The order of the spawners in the user guide has been updated to match the order of the actors in Unreal Engine. Makes it easier for users to follow along.

File: docs/user-guide/docs-source/spawn-in-volume.md [hunk 2]
```diff
@@ -1,10 +1,8 @@
...
+**Spawn in Volume** will use the <number-badge>3</number-badge> _density_ range and the <number-badge>5</number-badge> _rotation_ range you provide to instantiate copies of an Unreal Engine actor randomly selected from an <number-badge>7</number-badge> Array of Unreal Engine actors you've chosen within the box component's bounds. If you just want all spawned obstacles to face the same direction, you can select <number-badge>4</number-badge> _Restrict to One Rotation_ and **Spawn in Volume** will automatically update the range's maximum to equal its minimum. To enable physics simulation on your spawned obstacles, just select <number-badge>6</number-badge> Add Physics. By default, the **Spawn in Volume** will remove overlaps, but this behavior can be turned off by unselecting <number-badge>8</number-badge> Remove Overlaps. To change the placement pattern, just adjust the <number-badge>9</number-badge> _random seed_ value.
```
Comment (kris-work) [inline docs/user-guide/docs-source/spawn-in-volume.md:5]: "...Array of Unreal Engine actors...". "Array" is capitalized but "actors" isn't. We should be consistent. I suggest user lower case in most cases unless referring to a specific subclass type. So change "Array" -> "array" here.

Comment (kris-work) [reply docs/user-guide/docs-source/spawn-in-volume.md:5]: In most places, when referring to a param by it's label test we've been italicizing that text. Ex. "_Restrict to One Rotation_". But "Add Physics" hasn't been italicized here. Please italicize it both for consistency and better readability. Same applies to "Remove Overlaps".

Comment (kris-work) [reply docs/user-guide/docs-source/spawn-in-volume.md:5]: Capitalize "random seed" (number 9) to be consistent with other callouts.

File: docs/user-guide/docs-source/spawn-on-surface.md [hunk 3]
```diff
@@ -1,10 +1,8 @@
...
...</number-badge> surface name pattern <number-badge>2</number-badge> and/or by tag (see [How To Use Tags](/spawner-parent/#how-to-use-tags) below). When your simulation starts, the **Spawn on Surface** instantiates copies of an Unreal Engine actor randomly selected from an <number-badge>7</number-badge> Array of Unreal Engine actors you've chosen onto those surfaces in a random pattern at random angles using the <number-badge>3</number-badge> _density_ range and the <number-badge>5</number-badge> _rotation_ range you provide. If you just want all spawned obstacles to face the same direction, you can select <number-badge>4</number-badge> _Restrict to One Rotation_ and **Spawn on Surface** will automatically update the range's maximum to equal its minimum. To enable physics simulation on your spawned obstacles, just select <number-badge>6</number-badge> Add Physics. By default, the **Spawn on Surface** will remove overlaps, but this behavior can be turned off by unselecting <number-badge>8</number-badge> Remove Overlaps. A <number-badge>9</number-badge> _random seed_ value drives the random placement pattern. Don't like the pattern? Just enter a different random seed to try a new one.
```
Comment (kris-work) [inline docs/user-guide/docs-source/spawn-on-surface.md:5]: All the same consistency issues I called out for the `spawn-in-volume.md` should be applied here as well.

**Comment by kris-work** [comment]
Nit: `images/spawn-with-houdini/AmbitSpawnerForHoudini_UI_Annotated.png` doesn't fully match the style of other image annotations. The wrong font is used, font size relative to circle size is off, lines have rounded ends instead of square.

**Comment by kris-work** [comment]
Nit: `/images/spawn-vehicle-path/AmbitVehiclePathGenerator_UI.png`: Annotations have visual artifacts that the other images don't. Specifically, aliasing of the circles. Also, the numbers in the circles are the wrong weight and aren't all centered well.

**Comment by kris-work** [comment]
Images in `individual-scenario-generation.md`:

- Are red instead of our standard pink/purple
- Have square corners instead of rounded

**Comment by kris-work** [comment]
Images in `bulk-senario-generation.md`:

Images in `individual-scenario-generation.md`:

- Are red instead of our standard pink/purple
- Have square corners instead of rounded
- Appear to be a different weight than used elsewhere

**Comment by kris-work** [comment]
The file `bulk-senario-generation.md` has a typo in its name. It should be `bulk-scenario-generation.md`.

**Review by kris-work — CHANGES_REQUESTED** [review]
I'm going to insist on the highest standards here. Anything I've marked "nit" is at your discretion. But I'd like to see everything else addressed.

**Review by rukekre — COMMENTED** [review]
All concerns have been addressed. More images have been changed to increase consistency in the user guide.

**Review by kris-work — APPROVED** [review]

**Pull request merged**